querier_examples_test.go

```diff
@@ -81,6 +81,20 @@ func ExampleQuerier_WithTag() {
 	// Name: `Vicious Baron` (string), ID: `baron` (string), Start: 2014-06-01 00:00:00 +0000 UTC (time.Time), End: 2016-02-21 00:00:00 +0000 UTC (*time.Time)
 }
 
+func ExampleQuerier_WithQualifiedViewName() {
+	_, err := DB.WithQualifiedViewName("people_0").FindByPrimaryKeyFrom(PersonTable, 1)
```

Comment:
invalid operation: DB (variable of type *gopkg.in/reform.v1.DB) has no field or method WithQualifiedViewName